linear_x_axis : 1
linear_y_axis : 0
linear_z_axis : 4
angular_z_axis : 3
linear_x_axis_reverse : false
linear_y_axis_reverse : false
linear_z_axis_reverse : false
angular_z_axis_reverse : false
takeoff_button : 9
landing_button : 8